### ISSUE TITLE
[Feature request] html to text

### ISSUE THREAD

**Issue by rEnr3n** [EdJoPaTo/website-stalker#13]
I want to watch an html page. Currently, commit shows changes including html tags. I'm not interested in html tags. I'd like an option to filter them out.

**Comment by EdJoPaTo** [comment]
The syntax in v0.8 will be like this:
```yaml
  - url: "https://edjopato.de/post/"
    extension: html
    editors:
      - css_selector: h1 a
      - html_text
```

**Comment by EdJoPaTo** [comment]
`html_markdownify` is available with v0.10 and provides a better experience for me. It can be a drop in replacement for `html_textify`.
